website/content/docs/secrets/pki/rotation-primitives.mdx

```diff
@@ -261,6 +261,28 @@ cert and its `ca_chain` value.
    name parameters from the existing issuer; it merely reuses the same key
    material.
 
+##### Notes on `manual_chain`
+
+If an intermediate is cross-signed and imported into the same mount as its
+pair, Vault will not detect the cross-signed pairs during automatic chain
+building. As a result, leaf issuance will have a chain that only includes
+one of these pairs of chains. This is because leaf issuance's `ca_chain`
```

Comment:
Grammar question: Should this be `This is because a leaf issuance's` (or `the` leaf...) ?

Reply:
I think perhaps `the` would be best...?